tests/test_downloads.py

```diff
@@ -143,9 +143,7 @@ def test_download_cached(self, export, get_cached):
         assert not export.delay.called
 
     def test_download_forbidden(self):
-        # should the tests throw exceptions?
-        # with pytest.raises(ApiError):
-        with self.assertRaises(Exception):
```

Comment:
This is again a candidate for `git rebase -i`. Otherwise, we are confusing future maintainers by modifying and reverting code within the same PR.